flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -179,14 +185,17 @@ public boolean databaseExists(String databaseName) throws CatalogException {
   @Override
   public void createDatabase(String name, CatalogDatabase database, boolean ignoreIfExists)
       throws DatabaseAlreadyExistException, CatalogException {
+    createDatabase(name, mergeComment(database.getProperties(), database.getComment()), ignoreIfExists);
+  }
+
+  private void createDatabase(String databaseName, Map<String, String> comments, boolean ignoreIfExists)
+      throws DatabaseAlreadyExistException, CatalogException {
     if (asNamespaceCatalog != null) {
       try {
-        asNamespaceCatalog.createNamespace(
-            toNamespace(name),
-            mergeComment(database.getProperties(), database.getComment()));
+        asNamespaceCatalog.createNamespace(toNamespace(databaseName), comments);
```

Comment:
Should this be `metadata` instead of `comments`? I think that it could contain other properties, right?

Reply:
Make sense.